src/components/shared/partials.jsx

```diff
@@ -128,12 +140,12 @@ export const Synonyms = ({ synonyms }) =>
         {synonyms.map((s, index) => (
           <li
             style={{ paddingLeft: '10px' }}
-            key={s.genus + s.species + s.subSpecies + s.authority}
```

Comment:
you don't need to update the key -- the goal is just for it to be something unique that identifies it, which it will be without the notes